ros2_control_demo_hardware/src/rrbot_system_multi_interface.cpp

```diff
@@ -237,8 +239,8 @@ return_type RRBotSystemMultiInterfaceHardware::start()
   status_ = hardware_interface::status::STARTED;
 
   RCLCPP_INFO(
-    rclcpp::get_logger("RRBotSystemMultiInterfaceHardware"), "System successfully started! %u",
-    control_lvl_[0]);
+    rclcpp::get_logger("RRBotSystemMultiInterfaceHardware"), "System successfully started! %hhu",
+    static_cast<std::uint8_t>(control_lvl_[0]));
```

Comment:
Is there any other solution then adding `static_cast`? This doesn't look nice in the code.